plugins/outputs/yandex_cloud_monitoring/yandex_cloud_monitoring.go

```diff
@@ -0,0 +1,276 @@
+package yandex_cloud_monitoring
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/selfstat"
+)
+
+// YandexCloudMonitoring allows publishing of metrics to the Yandex Cloud Monitoring custom metrics
+// service
+type YandexCloudMonitoring struct {
+	Timeout            internal.Duration
+	EndpointUrl        string `toml:"endpoint_url"`
+	MetadataTokenUrl   string `toml:"metadata_token_url"`
+	MetadataFolderUrl  string `toml:"metadata_folder_url"`
+	Service            string `toml:"service"`
```

Comment:
`MetadataTokenUrl`, `MetadataFolderUrl` and `Service` are undocumented in sample config. Is this desired behavior?

Reply:
Yes. Non-default service can be set only in internal use cases and we plan to use it,
though for public clients it should always be set to default value "custom".
This is stated in docs.